lms/djangoapps/grades/api/views.py

```diff
@@ -1,24 +1,29 @@
 """ API v0 views. """
 import logging
 
+from django.contrib.auth import get_user_model
 from django.http import Http404
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 from rest_framework import status
 from rest_framework.authentication import SessionAuthentication
+from edx_rest_framework_extensions.authentication import JwtAuthentication
```

Comment:
Oo nice to know that exists!  I'll use it on my other PR too.  Addressed with 25d1586.